iset.mm

```diff
@@ -139377,6 +139377,19 @@ reduced fraction representation (no common factors, denominator
       YNYMHUEUGYDYAYEYFYG $.
   $}
 
+  ${
+    $d N j x $.
+    $( A natural number has finitely many divisors.  (Contributed by Jim
+       Kingdon, 9-Oct-2025.) $)
+    dvdsfi $p |- ( N e. NN -> { x e. NN | x || N } e. Fin ) $=
```

Comment:
This may be useful in set.mm, too.

Reply:
The proof in set.mm is a lot shorter, but, sure, it seems to shorten several proofs to add it, so I'll do that.